Javascript/homework/responses/2.1-css-linking.md

```diff
@@ -14,10 +14,19 @@ This week, we're going to take the HTML and JavaScript code we wrote last week a
 
 - Next you're going to open Visual Studio Code then at the top left click file, click Open folder then find your java script folder which then you're going to open week 2.
 
-To style our webpage, we must link a css file to our HTML file.  
+Once were in Visual Studio Code we're going to need to style our webpage, First we must link a CSS file to our HTML file.  
 
 - Add a `link` tag inside the head of your html file
   - Set the `rel` attribute to `"stylesheet"` and the `href` attribute to `"style.css"`.
+ <details><summary>Having Issues? Click Me!</summary>
+<p>
+
+```html
+<link rel="stylesheet" href="styles.css"
```

Comment:
I've noticed that a lot of times we give code for students as hints if they're struggling, so I would recommend always adding a couple lines explaining what that means even if it's something as simple as this.

Chances are, if they had to click on this to get the line of code they don't fully understand the concept so you might need to reiterate so they don't just copy and move on without understanding why.